templates/core/_init_tabs.html

```diff
@@ -2,11 +2,20 @@
 {# tab id must be {{ tabs }} (use {% with...%}) #}
 {% load i18n %}
 <script>
-
+const getCookieByName = (cookieName) => document.cookie.split('; ')
```

Comment:
this detection code is not really related to tabs init... We should define if it's only used for timesheet view (and hence be in timesheet page) or more global to pydici. But this init_tabs file is only intended for pages with... tabs ;-)

Reply:
I moved it to `templates/core/pydici.html`. if not the right place, I would need your suggestions :pray: